content/en/docs/started/installing-kubeflow.md

```diff
@@ -129,7 +129,7 @@ The following table lists <b>active distributions</b> that have <b>had a recent
         <td>Charmed Kubeflow</td>
         <td>Canonical</td>
         <td>
-          All Conformant Kubernetes
+          All Certified Kubernetes Distributions
```

Comment:
Certified by whom? Any references?

Reply:
@chensun I am referring to the CNCF Kubernetes Certification program: 
- https://www.cncf.io/certification/software-conformance/

I can add that as a link if you like?

Reply:
That would be ideal.

Reply:
@chensun done. 

I have linked to the official list on the Kubernetes website: https://kubernetes.io/partners/#conformance